connectors/opa/connector_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/stretchr/testify/assert"
 
+	app "fybrik.io/fybrik/manager/apis/app/v1alpha1"
```

Comment:
This does not seem right. Common definitions should be defined in pkg/model/... and be used by different components.